keps/sig-api-machinery/20190228-priority-and-fairness.md

```diff
@@ -963,50 +1187,44 @@ of this subsystem.  To accomplish these things there are two levels of
-objects in the implementation, but are figments of our imagination
-used to describe the behavior of this subsystem.  These backstop
-objects are implicitly present and affecting behavior when needed.
-There are two implicitly generated RequestPriority backstop objects.
-One is equivalent to the `system-top` object exhibited above, and it
-exists while there is no actual RequestPriority object with `exempt ==
-true`.  The other is equivalent to the `workload-low` object exhibited
-above, and exists while there is no RequestPriority object with
-non-exempt priority.  There are also two implicitly generated
-FlowSchema backup objects.  Whenever a request whose groups include
-`system:masters` is not matched by any actual FlowSchema object, a
-backstop equivalent to the `system-top` object exhibited above is
-considered to exist.  Whenever a request whose groups do not include
-`system:masters` is not matched by any actual FlowSchema object, the
-following backstop object is considered to exist.
+The effective configuration is the union of (a) preserving configuration 
+objects and (b) all the exhibited default configurations above. The 
+request-management system uses the preserving configuration as its initial 
+state on launching when not all the default configuration present in the 
+apiserver's storage and won't reload until ensuring/observing the preserving 
+objects exists. The default configurations will always be created everytime 
+an kube-apiserver restarts.
+
+Additionally, if an administrator disagrees with the default settings and
+feel like replacing with a complete new set. To avoid re-creation of the  
+system defaults above,  the administrators have to update the undesired 
+objects to have no effects
+
+Administrators who wish to operate with a different configuration can create, 
+update, and delete additional configuration objects and edit the predefined 
+ones; the predefined ones can not be deleted but they can be modified to 
+minimize their effects. In particular, to minimize the effects of a FlowSchema,
+you can modify to match no requests. To minimize the effects of a 
+PriorityLevelConfiguration modify AssuredConcurrencyShares=1. For example:
 
-```yaml
+```
 kind: FlowSchema
 meta:
-  name: non-top-backstop
+  name: matches-nothing
 spec:
-  matchingPriority: (doesn’t really matter)
-  requestPriority:
-    name: (name of an effectively existing RequestPriority, whether
-           that is real or backstop, with catchAll==true)
-  flowDistinguisher:
-    source: user
-    # no transformation in this case
-  match:
-  - and: [ ] # match everything
+  matchingPrecedence: 20000 # either logically lower than "default" flow-schema which catches all requests
+  priorityLevelConfiguration:
+    name: default
+  rules:  
+  - rule:
+      verbs: ['*']
+      apiGroups: ['*']
+      resources: ['*']
+    subjects:
+    - kind: User
+      name: "null" # or assign it any unreachable username
```

Comment:
Are there any invalid usernames?
Looking at the validation now, I see gaps in validation of AIPGroup, Resources, and NonResourceURLs.